packages/twenty-front/src/modules/auth/sign-in-up/components/SignInUpForm.tsx

```diff
@@ -127,34 +140,150 @@ export const SignInUpForm = () => {
         {authProviders.google && (
           <>
             <MainButton
-              Icon={() => <IconGoogle size={theme.icon.size.lg} />}
+              Icon={() => <IconGoogle size={theme.icon.size.md} />}
               title="Continue with Google"
               onClick={signInWithGoogle}
               fullWidth
             />
-            <HorizontalSeparator visible={!authProviders.microsoft} />
+            <HorizontalSeparator visible={false} />
```

Comment:
I think this is not right! why do we need a separator between google and microsoft SSO? and then why adding the component if visible = false?

Reply:
There is only one line between the SSO and Classic signin.
But we want a space without a visible line between SSO buttons. That's why visible is false.